src/System.Net.Security/tests/FunctionalTests/TestConfiguration.cs

```diff
@@ -14,7 +14,7 @@ namespace System.Net.Security.Tests
 {
     internal static class TestConfiguration
     {
-        public const int PassingTestTimeoutMilliseconds = 1 * 60 * 1000;
+        public const int PassingTestTimeoutMilliseconds = 4 * 60 * 1000;
```

Comment:
That's what I'm looking to find out. They don't fail on my machines, so I'm increasing it to a value that will remove the question of whether it is a issue of other tests hogging the machine or if they're failing because of some legitimate issue.